package.json

```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "lint:report": "next lint --output-file eslint_report.json --format json",
```

Comment:
This script runs the same linter but it outputs in json rather than in the terminal. 

This is so that we can later parse that file and write GitHub annotations for each error and warning.